WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
No, I'm thinking `"seconds": 1.23`. I'm saying the `1.23` part is a string, since JSON _is_ one big string, even though this _is_ the JSON encoding of a number. Since JSON encodes to decimal strings, there is no binary/decimal friction in JSON.

OK, "ugly" is a subjective opinion, but an objective fact is that "centisecond", while a valid SI unit, is a very rare word that most people never use and probably will have to look up. Nobody says the World Record is 473 centiseconds.